src/settings/settings.lua

```diff
@@ -0,0 +1,16 @@
+return {
+    leftRightSpeedPercent = 50,
+    softDropSpeedPercent = 50,
+    delayedAutoShiftPercent = 50,
+    gravitySpeedPercent = 50,
+
+    keyBindings = {
+        ['moveLeft'] = {'left'},
+        ['moveRight'] = {'right'},
+        ['moveDown'] = {'down'},
+        ['rotateCw'] = {'up'},
+        ['rotateCcw'] = {'z'},
+        ['hardDrop'] = {'space'},
+        ['hold'] = {'lshift', 'rshift', 'c'}
+    }
```

Comment:
this whole part can be removed.

Reply:
Okay if you want to keep it can you change the order? "Key to press" = "action"?